### PR TITLE
update fitted params for naive_bayes and linear discriminant analysis

### LINKED ISSUE #53: `fitted_params` for GaussianNBClassifier broken
```julia
X, y = MLJBase.make_moons()
mach = MLJBase.machine(GaussianNBClassifier(), X, y)
MLJBase.fit!(mach)
MLJBase.fitted_params(mach)

ERROR: KeyError: key :sigma_ not found
Stacktrace:
 [1] __getproperty
   @ ~/.julia/packages/PyCall/twYvK/src/PyCall.jl:313 [inlined]
 [2] getproperty(o::PyCall.PyObject, s::Symbol)
   @ PyCall ~/.julia/packages/PyCall/twYvK/src/PyCall.jl:318
 [3] fitted_params(m::GaussianNBClassifier, ::Tuple{PyCall.PyObject, CategoricalArrays.CategoricalValue{Int64, UInt32}, Nothing})              
   @ MLJScikitLearnInterface ~/MLJ/MLJScikitLearnInterface/src/models/misc.jl:57
 [4] fitted_params(mach::MLJBase.Machine{GaussianNBClassifier, true})
   @ MLJBase ~/.julia/packages/MLJBase/97P9U/src/machines.jl:821
 [5] top-level scope
   @ REPL[38]:1
```

### PR DESCRIPTION
**Pull request by OkonSamuel**
closes #53 

cc. @ablaom

### CODE REVIEW

**Comment by ablaom** [comment]
@OkonSamuel Thanks for the prompt attention.

Can you please remove [this line](https://github.com/JuliaAI/MLJScikitLearnInterface.jl/blob/9d9f4dca8fa0187d49545eb014b1df775f562f85/test/generic_api_tests.jl#L52) and the one below to revert testing of the NB model?

File: src/models/ensemble.jl [hunk 1]
```diff
@@ -7,6 +7,7 @@ const AdaBoostRegressor_ = sken(:AdaBoostRegressor)
     random_state::Any      = nothing
 end
 MMI.fitted_params(model::AdaBoostRegressor, (f, _, _)) = (
+    estimator            = f.estimator_,
     estimators           = f.estimators_,
     estimator_weights    = f.estimator_weights_,
```
Comment (ablaom) [inline src/models/ensemble.jl:12]: Do the python models really support both `estimator` and `estimators`?

File: src/models/ensemble.jl [hunk 2]
```diff
@@ -260,7 +266,7 @@ const ExtraTreesRegressor_ = sken(:ExtraTreesRegressor)
     min_samples_leaf::Union{Int,Float64}   = 1::(_ > 0)
     min_weight_fraction_leaf::Float64      = 0.0::(_ ≥ 0)
     ## TODO: Remove the "auto" option in python sklearn v1.3
-    max_features::Union{Int,Float64,String,Nothing} = "sqrt"::(_ === nothing || (isa(_, String) && (_ in ("auto","sqrt","log2"))) || _ > 0)
+    max_features::Union{Int,Float64,String,Nothing} = 1.0::(_ === nothing || (isa(_, String) && (_ in ("auto","sqrt","log2"))) || (_ isa Number && _ > 0))
     max_leaf_nodes::Option{Int}    = nothing::(_ === nothing || _ > 0)
```
Comment (ablaom) [inline src/models/ensemble.jl:270]: This is a breaking change and we already tagged a breaking release. I guess we consider this a "bug fix"?

File: src/models/svm.jl [hunk 3]
```diff
@@ -28,7 +29,7 @@ const SVMClassifier_ = sksv(:SVC)
     C::Float64          = 1.0::(_ > 0)
     kernel::Union{String,Function}  = "rbf"::(_ isa Function || _ in ("linear", "poly", "rbf", "sigmoid", "precomputed"))
     degree::Int         = 3::(_ > 0)
-    gamma::Union{Float64,String}    = "auto"::(_ isa Float64 && _ > 0 || _ in ("scale", "auto"))
+    gamma::Union{Float64, String}    = "scale"::((_ isa Float64 && _ > 0) || _ in ("scale", "auto"))
     coef0::Float64      = 0.0
```
Comment (ablaom) [inline src/models/svm.jl:33]: Similarly here.

**Review by ablaom — APPROVED** [review]
Looks good. Thanks for attention to some previously overlooked detail. 

I'm inclined to make this a breaking release, to be safe, because of the default value changes. But I would like to be sure there aren't any more we have missed. What do you think?

**Comment by ablaom** [comment]
@OkonSamuel Maybe we missed a hyperparameter name change: https://github.com/JuliaAI/MLJScikitLearnInterface.jl/pull/56#discussion_r1122300818 ??

File: src/models/ensemble.jl (hunk 1 above)
Comment (OkonSamuel) [reply src/models/ensemble.jl:12]: Yes, they do

File: src/models/ensemble.jl (hunk 2 above)
Comment (OkonSamuel) [reply src/models/ensemble.jl:270]: We haven't released a breaking release yet

**Comment by OkonSamuel** [comment]
> @OkonSamuel Maybe we missed a hyperparameter name change: [#56 (comment)](https://github.com/JuliaAI/MLJScikitLearnInterface.jl/pull/56#discussion_r1122300818) ??

No, not really. It's actually a deprecation I left there. I plan on removing it when python sklearn releases v1.4

**Pull request merged**